eras/babbage/impl/src/Cardano/Ledger/Babbage/Rules/Utxow.hs

```diff
@@ -364,7 +379,7 @@ instance
   type BaseM (BabbageUTXOW era) = ShelleyBase
   type PredicateFailure (BabbageUTXOW era) = BabbageUtxowPredFailure era
   type Event (BabbageUTXOW era) = AlonzoUtxowEvent era
-  transitionRules = [babbageUtxowTransition]
+  transitionRules = [babbageUtxowMirTransition >> babbageUtxowTransition @era]
```

Comment:
Not sure if this matters, but by defining it this way, the order that the Predicate tests run now, is not the same as the order they used to be run.  If it doesn't matter, we should say so, and explain why.

Reply:
The important part is that predicate is checked. The order only controls the order in which the user sees the failure, so that is not that important. This is a general rule, which doesn't require a special documentation in this case.

In this case especially it doesn't matter, since MIR certificates are can only be submitted by IOG.

Reply:
Forgot to mention, although I think it is obvious, but the order matters for any state modification. So, if `babbageUtxowMirTransition` did modify the state in a way that mattered for the order of computation, then clearly such such re-ordering would not be possible. But, this is not the case since it only contains a predicate check and does not affect the state in any way